src/sidecar/sidecarManager.ts

```diff
@@ -379,17 +379,54 @@ export class SidecarManager {
         // Set up the environment for the sidecar process.
         const sidecar_env = constructSidecarEnv(process.env);
 
+        const stderrPath = `${SIDECAR_LOGFILE_PATH}.stderr`;
         try {
+          // try to create a file to track any stderr output from the sidecar process
+          fs.writeFileSync(stderrPath, "");
+          const stderrFd = fs.openSync(stderrPath, "w");
+
           const sidecarProcess = spawn(executablePath, [], {
             detached: true,
-            stdio: "ignore",
+            // ignore stdin/stdout, pipe stderr to the file
```

Comment:
```suggestion
            // ignore stdin/stdout, stderr to the file
```

NodeJS isn't "piping" anything. file descriptor 2 for the new process will be attached to the file as if it were open()ed for writing by the process before main() starts.